README.md

```diff
@@ -1,19 +1,49 @@
 # Brooklyn CLI
 
-## Compiling
-
-1. Ensure your [$GOPATH](http://golang.org/cmd/go/#hdr-GOPATH_environment_variable) is set correctly,
-   to some location where Go does its work, such as `~/gocode` .
-2. Get and build the cli source code: `go get github.com/brooklyncentral/brooklyn-cli/br`
-3. Run it from `$GOPATH/bin/br` (add `$GOPATH/bin/` to your PATH).
-4. Thereafter if you want to do code changes, 
-   link the `$GOPATH/src/github.com/brooklyncentral/brooklyn-cli`
-   with the directory where you want to keep your git repositories.
-   (TODO: clarify best practice for this, including how to combine
-   it with a Brooklyn all-projects build)
+## Toolchain
+
+You will need the following tools to build the CLI:
+- Go (min version 1.5.1), with full cross-compiler support (see https://golang.org/doc/install/source).
```

Comment:
For this Toolchain section, maybe better to just provide a high level link (eg https://golang.org/doc/) rather than jumping to install info.  So keep it descriptive of the tools.